build/attestations/index.md

```diff
@@ -139,7 +139,7 @@ attestation.
 
 <!-- prettier-ignore -->
 To deep-dive into the specifics about how attestations are stored, see
-[Image Attestation Storage (BuildKit)](https://github.com/moby/buildkit/blob/master/docs/attestation-storage.md){: target="blank" rel="noopener" class="_"}.
+[Image Attestation Storage (BuildKit)](https://docs.docker.com/build/attestations/attestation-storage/){: target="blank" rel="noopener" class="_"}.
```

Comment:
Should have been a relative link: `[Image Attestation Storage (BuildKit)](attestation-storage.md)` without target blank as well.